test/Scheduling/chaining-cyclic-problems.mlir

```diff
@@ -0,0 +1,167 @@
+// RUN: circt-opt %s -ssp-roundtrip=verify
+// RUN: circt-opt %s -ssp-schedule="scheduler=simplex options=cycle-time=5.0" | FileCheck %s -check-prefixes=CHECK,SIMPLEX
+
+
+
+// test from cyclic-problems.mlir
```

Comment:
Good idea to include the parent problem's tests here 👍 Might make sense to think about a mechanism to do this without duplication (in the future / not in this PR).